src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -516,6 +515,10 @@ ov::AnyMap get_default_prefill_config(const std::shared_ptr<ov::Model>& model,
     auto config = get_default_common_config(model);
     if (is_cw_compressed(model)) {
         config.emplace("NPUW_DQ", "YES");
+        if (npudesc.has_value() && npudesc->compiler_dq) {
+            config.emplace("NPUW_DQ_FULL", "NO");
+            config.emplace("NPU_COMPILER_DYNAMIC_QUANTIZATION", true);
+        }
```

Comment:
Why do you enable it for CW compressed models only? The whole point of this feature was to make it for prefill.

Reply:
Done